Tooll/Components/TextButtonEdit.xaml.cs

```diff
@@ -125,6 +132,14 @@ protected void TextEdit_LostFocus(object sender, RoutedEventArgs e)
             }
             TextEdit.Visibility = Visibility.Collapsed;
         }
+        
+        protected void TextEdit_PreviewMouseDown(object sender, MouseButtonEventArgs e)
+        {
+            if (e.ClickCount == 3)
+            {
+                XTextEdit.SelectAll();

```

Comment:
another option to handle this would be to add a timer myself. but i thought that would be too line-heavy. i actually took a while with debugging and system.diagnostic to actually find out, that the TextBox is buggy and always returns a `e.ClickCount == 1`, if you do not use the PreviewHandler >_< and i think (not sure about this!) that the `e.Handled = true;` would be redundant. it should be handled in the `System.Windows.Control`